uint32 shape_type_code
---
std_msgs/Bool is_possible
